front/style/layout/_nav.scss

```diff
@@ -189,3 +189,19 @@
         }
     }
 }
+
+.gtm__navigation__mobile {
+
+    .btn {
+        @include btn;
```

Comment:
That's redundant here. The class `btn` by default has the `@include btn`.  That's why it's better to have more specific names. So here the class `btn` should be more specific and use the BEM standards.

Reply:
So you can have in the template `btn` `btn--pink`. All this CSS actually should be unnecessary. I can explain better in person!